<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1393</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Рабочее окно</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,1,3,3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;
</string>
         </property>
         <property name="text">
          <string>Исходные ячейки</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,1">
         <item>
          <widget class="QLabel" name="original">
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(241, 241, 241);
border-radius: 13px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="original_clear">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="10,1">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;</string>
         </property>
         <property name="text">
          <string>Команда</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="command">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Разъединить</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Переименовать</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Объединить</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;</string>
         </property>
         <property name="text">
          <string>Итоговые ячейки</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,1">
         <item>
          <widget class="QLabel" name="result">
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(241, 241, 241);
border-radius: 13px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="result_clear">
           <property name="text">
            <string>X</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="apply_btn">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: 'Gilroy';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 20px;
text-align: center;

color: #FFFFFF
;background: #575757;
border-radius: 15px;</string>
       </property>
       <property name="text">
        <string>применить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="5,1">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="styleSheet">
           <string notr="true">background-color: rgb();
font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;</string>
          </property>
          <property name="text">
           <string>Загруженная таблица</string>
          </property>
         </widget>
         <widget class="QTableView" name="table_before"/>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">
background-color: rgb();
font-family: 'Gilroy';
font-style: normal;
font-weight: 400;
font-size: 19px;
line-height: 23px;

color: #000000;</string>
          </property>
          <property name="text">
           <string>Итоговая таблица</string>
          </property>
         </widget>
         <widget class="QTableView" name="table_after"/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="change_file_btn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #575757;
border-radius: 15px;

font-family: 'Gilroy';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 20px;
text-align: center;

color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>выбрать другой файл</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="go_to_download_btn">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background: #575757;
border-radius: 15px;

font-family: 'Gilroy';
font-style: normal;
font-weight: 600;
font-size: 16px;
line-height: 20px;
text-align: center;

color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>сохранить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
